### PR TITLE
Release automation - Github

### PR DESCRIPTION
**Pull request by abu-usama**
created workflow for github release

### CODE REVIEW

File: cidr2ip/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## cidr2ip-1.0.0
```
Comment (UmanShahzad) [inline cidr2ip/CHANGELOG.md:1]: Is it safe for us to add an extra newline below this? I usually create separations like that.

Also, do we have to use two of the hash characters or can we go with 1 as well for these for detection via `changelog.sh`?

File: scripts/build-github-actions.sh
Comment (UmanShahzad) [inline scripts/build-github-actions.sh:1]: If this script is the same as `scripts/release.sh` but without the github release push, then let's deduplicate by creating a `scripts/build-all.sh` script which is used by github actions and also used by `./scripts/release.sh`.

File: scripts/build-github-actions.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# NOTE: Github actions uses this script.
```
Comment (UmanShahzad) [inline scripts/build-github-actions.sh:3]: We can rm this comment after the refactor mentioned above

**Review by UmanShahzad — COMMENTED** [review]
- Are we not auto-uploading the PPA? Will we do that in the next PR?
- How about auto-push of the new Docker? Next PR?

File: cidr2ip/CHANGELOG.md (hunk 1 above)
Comment (abu-usama) [reply cidr2ip/CHANGELOG.md:1]: Yes for both.
I'll update the `changelog.sh` accordingly.

**Comment by abu-usama** [comment]
I'll be creating new PRs for docker & PPA releases.

File: scripts/changelog.sh [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+DIR=`dirname $0`
+ROOT=$DIR/..
+
+CLI=$1
+VERSION=$2
+
+found=0
+
+cat $ROOT/${CLI}/CHANGELOG.md | while read "line"; do
+
```
Comment (UmanShahzad) [inline scripts/changelog.sh:17]:
```suggestion
```

**Review by UmanShahzad — APPROVED** [review]

**Pull request merged**